src/setupTests.ts

```diff
@@ -3,3 +3,6 @@
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
 import '@testing-library/jest-dom';
+import { toHaveNoViolations } from 'jest-axe';
```

Comment:
is there a reason you're just adding the one `expect` method? Not that we need more right now but if we ever did in the future

Reply:
I think that's the only one from `jest-axe`